code-challenges.test.js

```diff
@@ -56,4 +120,25 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
+// going to make another describe statement with an it statement to describe the code and an expect statement to  call the function.
+
+describe("vowelIndex", () => {
+    it("takes in a string and logs the index of the first vowel.", () => {
+        let expectedIndex = [1, 0, 2]
+      expect(vowelIndex(vowelTester1, vowelTester2, vowelTester3)).toEqual(expectedIndex)
+    })
+  })
+
 // b) Create the function that makes the test pass.
+// Now I am going to create a function with conditional if "a, e, i, o, u" then return index of
```

Comment:
Once again, there is almost no pseudocode to show the process of the code you have written.  This meets the bare minimum requirements for this section for pseudocode. Remember, it is not about having working code, but it IS about your process and how you problem-solve.  This is what I am most interested in.